api/schema_system.go

```diff
@@ -53,6 +56,52 @@ const (
 
 	// SchemaRuleSchemaBackwardCompatibility enforce the MySQL and TiDB support check whether the schema change is backward compatible.
 	SchemaRuleSchemaBackwardCompatibility SchemaReviewRuleType = "schema.backward-compatibility"
+
+	// TemplateRuleStringSection is the string type in TemplateRuleSection
+	TemplateRuleStringSection TemplateRuleSectionType = "string"
+	// TemplateRuleTemplateSection is the template type in TemplateRuleSection
+	TemplateRuleTemplateSection TemplateRuleSectionType = "template"
+
+	// TemplateBracketLeft is the left bracket for template
+	TemplateBracketLeft = "{{"
```

Comment:
We can consider taking this function if needed.

https://github.com/bytebase/bytebase/blob/main/api/project.go#L254

Reply:
Updated. Now the unmarshal function will return the raw format string like `"hard_code_{{table}}_{{column}}_end"`, and all the template keys `["{{table}}", "{{column}}"]`, we need to replace the key with the real data to build the regex in advisor. WDYT @rebelice ?

Reply:
SGTM